### PR TITLE
Update folding-at-home from 7.5.1 to 7.6.9

### PR DESCRIPTION
**Pull request by greveritt**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vladimyr** [comment]
@greveritt Change appcast to `https://foldingathome.org/alternative-downloads/` and try again. Also, please update the homepage to `https://foldingathome.org/`.

**Comment by greveritt** [comment]
@vladimyr They're updated now.

File: Casks/folding-at-home.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
 cask 'folding-at-home' do
-  version '7.5.1'
-  sha256 '94fd47ff55120c1c869d0dfbf366ae418406a3b47b0fa1bcf94014777abb72a6'
+  version '7.6.9'
+  sha256 '9475e8aef3614daf92e3cfc8748be5e10e1c5b38a5c5da30ce3fe60f754de16b'
 
   # download.foldingathome.org/releases/public/release/fah-installer was verified as official when first introduced to the cask
   url "https://download.foldingathome.org/releases/public/release/fah-installer/osx-10.11-64bit/v#{version.major_minor}/fah-installer_#{version}_x86_64.mpkg.zip"
-  appcast 'https://download.foldingathome.org/releases/public/release/fah-installer/'
+  appcast 'https://foldingathome.org/alternative-downloads/'
```
Comment (vladimyr) [inline Casks/folding-at-home.rb:7]: @greveritt Sorry to bug you again but it turns out that download options are actually rendered dynamically  using following script: https://download.foldingathome.org/js/fah-downloads.js

File: Casks/folding-at-home.rb [hunk 2]
```diff
@@ -1,12 +1,12 @@
 cask 'folding-at-home' do
-  version '7.5.1'
-  sha256 '94fd47ff55120c1c869d0dfbf366ae418406a3b47b0fa1bcf94014777abb72a6'
+  version '7.6.9'
+  sha256 '9475e8aef3614daf92e3cfc8748be5e10e1c5b38a5c5da30ce3fe60f754de16b'
 
   # download.foldingathome.org/releases/public/release/fah-installer was verified as official when first introduced to the cask
   url "https://download.foldingathome.org/releases/public/release/fah-installer/osx-10.11-64bit/v#{version.major_minor}/fah-installer_#{version}_x86_64.mpkg.zip"
```
Comment (vladimyr) [inline Casks/folding-at-home.rb:6]: Remove the comment above because it isn't needed anymore and it makes style check fail.

**Comment by greveritt** [comment]
@vladimyr @core-code Thank you so much for resolving this for me!

**Pull request merged**